packages/cli/src/playground/src/pages/workflows/workflow/v-next/traces.tsx

```diff
@@ -11,8 +11,9 @@ function WorkflowTracesPage() {
   const { workflowId } = useParams();
 
   const { vNextWorkflow, isLoading: isWorkflowLoading } = useVNextWorkflow(workflowId!);
+  const { traces, error, firstCallLoading } = useTraces(vNextWorkflow?.name || '', '', true);
```

Comment:
ℹ️ This thing will need cleaning and refactoring + moving to tanstack query. Will be done in another PR